stone/data_type.py

```diff
@@ -488,6 +488,35 @@ def _check_list_container(self, val):
             raise ValueError('list has fewer than %s item(s)' % self.min_items)
 
 
+class Map(Composite):
+    def __init__(self, key_data_type, value_data_type):
+        super(Map, self).__init__()
+
+        if not isinstance(key_data_type, String):
+            raise ParameterError("Only String primitives are supported as key types.")
+
+        self.key_data_type = key_data_type
+        self.value_data_type = value_data_type
+
+    def check(self, val):
+        raise NotImplementedError
+
+    def check_example(self, ex_field):
+        for k,v in ex_field.value.items():
```

Comment:
It looks like tests pass in CI. 👍 It's just failing on linter stuff. E.G., from https://travis-ci.org/dropbox/stone/jobs/243434091#L285: `stone/data_type.py:505:14: E231 missing whitespace after ','`.